CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+then a hard fork could be enacted that limits the number of DReps in a more restrictive way.
+
+Reasonable numbers for the initial value of $X$ are probably 5,000-10,000.
+Reasonable numbers for the initial value of $Y$ are probably the total number of Lovelace
+divided by the initial value of $X$.
+
+The mechanism should be set to relax at a rate where the restriction is completely eliminated after
+a period of six months to one year.
+
+#### Other Ideas / Open Questions
+
+##### Pledge-weighted SPO voting
+
+The SPO vote could additionally be weighted by each SPO's pledge.
+This would provide a mechanism for allowing those with literal stake in the game to have a stronger vote.
+The weighting should be carefully chosen.
+
+##### Automatic re-delegation of DReps
+
+A DRep could optionally list another DRep credential in their registration certificate.
+Upon retirement, all of the DRep's delegations would be automatically transferred to the
+given DRep credential.  If that DRep had already retired, the delegation would be transfer
+to the 'Abstain' DRep.
+
+##### No DRep registration
+
+Since the DRep registration does not perform any necessary functions,
+the certificates for (de-)registering DReps could be removed. This
+makes the democracy more liquid since it removes some bureaucracy and
+also removes the need for the DRep deposit, at the cost of moving the anchor that is part of the
+DRep registration certificate into the transaction metadata.
+
+##### Reduced deposits for some government actions
+
+The deposit that is attached to governance actions exists to prevent a flood of non-serious governance
+actions, each of which would require time and attention from the Cardano community.
+We could reduce this deposit for proposals which go through some agreed upon off-chain process.
+This would be marked on-chain by the endorsement of at least one constitutional committee member.
+The downside of this idea is that it gives more power to the constitutional committee.
+
+##### Include hash of (future) genesis configuration within hard-fork proposal
+
+Some hard-forks require new genesis configurations.
+This has been the case for the Shelley and Alonzo hard forks (but not Allegra, Mary, Vasil or Valentine), may be the case in the future.
+At the moment, this proposal doesn't state anything about such a genesis configuration:
+it is implicitly assumed to be an off-chain agreement.
+We could however, enforce that (the hash of) a specific genesis configuration is also captured within a hard-fork governance action.
+
+##### Renaming DReps / state of no-confidence?
+
+It has been stated several times that "DReps" as presented here, might be confused with Project Catalst DReps.
+Similarly, some people have expressed confusion between the state of no-confidence, the motion of no-confidence and the no-confidence DReps.
+
+We could imagine finding better terms for these concepts.
+
+##### Rate-limiting treasury movements
```

Comment:
Yes indeed protocol parameters should be established to rate limit treasury withdrawals. This should also instil a process of setting quarterly and yearly budgets to prevent unwanted exhaustion of funds.

This is all out of scope of 1694 so feel free to disregard: Constitution can make it so that a yearly budget is set by sitting CC at the start of the year and agreed upon by SPOs + dReps. This budget serves as a constraint for future committees and groups.